src/lib.rs

```diff
@@ -1281,6 +1295,52 @@ fn multiplexed_enum_variant_name(
     ))
 }
 
+fn render_embedded_can_trait(w: &mut impl Write, msg: &Message) -> Result<()> {
+    writeln!(
+        w,
+        "
+impl embedded_can::Frame for {0} {{
```

Comment:
Can we gate this with a feature flag?

Reply:
Added feature flag defaulting to `Always` as `embedded_can` crate dependency is required anyway for representing CAN frame IDs